ql/src/java/org/apache/hadoop/hive/ql/exec/vector/VectorGroupByOperator.java

```diff
@@ -85,6 +86,19 @@ public class VectorGroupByOperator extends Operator<GroupByDesc>
   private static final Logger LOG = LoggerFactory.getLogger(
       VectorGroupByOperator.class.getName());
 
+  private static final Map<String, Class<?>> PRIMITIVE_TYPE_NAME_TO_CLASS = new ImmutableMap.Builder<String, Class<?>>()
```

Comment:
Could we reuse `PrimitiveObjectInspectorUtils.getTypeNameFromPrimitiveJava(TYPE)`?

Reply:
Thanks @deniskuzZ, I managed to get rid of the map via `PrimitiveObjectInspectorUtils.getTypeEntryFromTypeName(typeName).primitiveJavaClass`